Linux/config.yml

```diff
@@ -0,0 +1,15 @@
+UseAltVoice: false
+ResponseConfig:
+  StartedTracking:
+    - Text: Hello. I was told to wear a tux, but I guess the instructions were unclear.
+      Weight: 2
+    - Text: Welcome to Linux text-to-speech. As per tradition, it's like 5 times more difficult to setup and only half as good.
+      Weight: 2
+    - Text: Compiling shaders. Please wait.
```

Comment:
This line is too good to leave in here, tbh.

Reply:
Hah.  I added it because of steam's preloading of shaders taking forever on Linux, but I can maybe add it to the regular config, but it'll maybe be more frequent here. lol